piksi_tools/action_logger.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+import piksi_tools.serial_link as sl
+from sbp.client.handler import Handler
+from sbp.logging import SBP_MSG_PRINT
+from sbp.tracking import MsgTrackingState
+from sbp.piksi import MsgMaskSatellite, SBP_MSG_RESET
+from sbp.system import SBP_MSG_HEARTBEAT
+from sbp.table import dispatch
+
+import time
+import sys
+import random
+import threading
+
+DEFAULT_POLL_INTERVAL = 60 # Seconds
+DEFAULT_MIN_SATS = 5 # min satellites to try and retain
+
+
+class LoopTimer(object):
+  """ Interval timer (emulated from from stack overflow)
```

Comment:
Can you be more consistent with putting the description on the second line in the docstring?